base-server/index.js

```diff
@@ -496,6 +505,10 @@ export class BaseServer {
       channel.regexp = pattern
     }
     this.channels.push(channel)
+
+    if (queue) {
+      this.queueChannels.set(pattern, queueChannel)
```

Comment:
Seems like you are copying the same `queueChannel` instance to every query.

Do we need to create a new instance here?

Reply:
Yes, that is an oversight on my part. I replaced object with constructor.